addons/stock_account/models/stock.py

```diff
@@ -19,15 +19,33 @@ class StockInventory(models.Model):
         help="Date at which the accounting entries will be created"
              " in case of automated inventory valuation."
              " If empty, the inventory date will be used.")
+    has_account_moves = fields.Boolean(
+        default=False, store=False,
+        compute='_compute_has_account_moves')
+
+    def _compute_has_account_moves(self):
+        for inventory in self:
+            inventory.has_account_moves = False
+            if inventory.state == 'done':
+                account_move = self.env['account.move'].search([('stock_move_id.id', 'in', inventory.move_ids.ids)])
+                if account_move.exists():
+                    inventory.has_account_moves = True
```

Comment:
i think we should take care with this kind of compute, it could be quite costly with the prefetching, the orm could go crazy when opening a single inv adj

you should probably get the move_ids for all inv ajd out of the loop and do a single query, read_group could help you here

Reply:
``` python
for inventory in self:
    account_moves = self.env['account.move'].search_count([
        ('stock_move_id.id', 'in', inventory.move_ids.ids)
    ])
    inventory.has_account_moves = account_moves > 0
```

Reply:
It's done, thanks !